src/strategies/sd-boost-twavp/index.ts

```diff
@@ -0,0 +1,186 @@
+import { multicall } from '../../utils';
+
+export const author = 'clement-ux';
+export const version = '0.0.1';
+
+const abi = [
+  'function balanceOf(address account) external view returns (uint256)',
+  'function totalSupply() external view returns (uint256)',
+  'function working_supply() external view returns (uint256)'
+];
+
+const veSDT = '0x0C30476f66034E11782938DF8e4384970B6c9e8a';
+
+const F1 = 0.4;
+const F2 = 0.6;
+
+const chunk = (arr, size) =>
+  Array.from({ length: Math.ceil(arr.length / size) }, (v, i) =>
+    arr.slice(i * size, i * size + size)
+  );
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+): Promise<Record<string, number>> {
+  // Maximum of 5 multicall
+  expect(options.sampleStep).toBeLessThan(6);
```

Comment:
Hi @clement-ux 

I get this error when I run this without jest.
```
ReferenceError: expect is not defined
```


should be like this:
https://github.com/snapshot-labs/snapshot-strategies/blob/master/src/strategies/erc721-with-tokenid-range-weights-simple/index.ts#L61

Reply:
Changes done :)